Resources/Maps/Shuttles/skipper.yml

```diff
@@ -16,11 +16,22 @@ tilemap:
 entities:
 - proto: ""
   entities:
+  - uid: 5
+    components:
+    - type: MetaData
+    - type: Transform
+    - type: Map
+    - type: PhysicsMap
+    - type: GridTree
+    - type: MovedGrids
+    - type: Broadphase
+    - type: OccluderTree
+    - type: LoadedMap
```

Comment:
needs to be saved as a grid, not a map

Reply:
Oh, i did save it as grid, but it adds those anyway